components/geometry-merger/README.md

```diff
@@ -20,19 +20,34 @@ still be manipulated individually since this component keeps a `faceIndex` and
 
 ### API
 
-For either `geometry-merger` or `buffer-geometry-merger` components:
```

Comment:
We now have just one component, geometry-merger, which handles merging of BufferGeometries.